packages/gateway/src/generated/openapi/request/index.ts

```diff
@@ -14,4 +14,4 @@
  * limitations under the License.
  */
 
-export type * from "./OpenApiRequest.js";
+export type { OpenApiRequest } from "./OpenApiRequest.js";
```

Comment:
Are these the 5.x features?

Reply:
The feature was `type *` was introduced in ts 5.